src/main/java/seedu/address/ui/DriverCard.java

```diff
@@ -35,7 +38,16 @@ public DriverCard(Driver driver, int displayedIndex) {
         name.setText(driver.getName().fullName);
         phone.setText("Phone: " + driver.getPhone().value);
         driverId.setText("Driver ID: #" + driver.getId());
-        availability.setText("Unavailable Time: " + driver.getSchedule());
```

Comment:
There's no need to change this line

Reply:
oh this was because of a merge conflict i think